README.md

```diff
@@ -1,306 +1,60 @@
-
-To disable kubemacpool MAC assignment on a specific namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
-namespace/example-namespace labeled
-```
-
-To re-enable kubemacpool MAC assignment in a namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocate mutatevirtualmachines.kubemacpool.io=allocate
-namespace/example-namespace labeled
-```
-
-**note:** If a VMI is created directly and not through a VM, then it is handled in kubemacpool by the pod handler.
-
-### Check deployment
-
-Configmap:
-```bash
-[root]# kubectl -n kubemacpool-system describe configmaps
-
-Name:         kubemacpool-mac-range-config
-Namespace:    kubemacpool-system
-Data
-====
-RANGE_END:
-----
-FD:FF:FF:FF:FF:FE
-
-RANGE_START:
-----
-02:00:00:00:00:11
-```
-
-pods:
-```bash
-kubectl -n kubemacpool-system get po
-NAME                                                  READY   STATUS    RESTARTS   AGE
-kubemacpool-mac-controller-manager-6894f7785d-t6hf4   1/1     Running   0          107s
-```
-
-### Example
-
-Create a network-attachment-definition:
-
-The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the pod.
-This is follows the [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit)
-to provide a standardized method by which to specify the configurations for additional network interfaces.
-This standard is put forward by the Kubernetes [Network Plumbing Working Group](https://docs.google.com/document/d/1oE93V3SgOGWJ4O1zeD1UmpeToa0ZiiO6LqRAmZBPFWM/edit).
+Check the under interfaces section:
 
 ```yaml
-apiVersion: "k8s.cni.cncf.io/v1"
-kind: NetworkAttachmentDefinition
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
 metadata:
-  name: ovs-conf
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: ovs-cni.network.kubevirt.io/br1
+  name: simplevm
+...
 spec:
-  config: '{
-      "cniVersion": "0.3.1",
-      "name": "ovs-conf",
-      "plugins" : [
-        {
-          "type": "ovs",
-          "bridge": "br1",
-          "vlan": 100
-        },
-        {
-          "type": "tuning"
-        }
-      ]
-    }'
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - macAddress: "02:00:00:00:00:00"
+            masquerade: {}
+            name: default
+      networks:
+      - name: default
+        pod: {}
+...
 ```
 
-This example used [ovs-cni](https://github.com/kubevirt/ovs-cni/).
+For more examples see [VirutalMachine Allocation Example Guide](doc/vm-creation-example.md) and [Pod Allocation Example Guide](doc/pod-creation-example.md)
 
-**note** the tuning plugin change the mac address after the main plugin was executed so
-network connectivity will not work if the main plugin configure mac filter on the interface.
+### Kubemacpool service
```

Comment:
Default behavior?

Service is pretty vague and Kubemacpool is the context of this whole readme

Reply:
perhaps I can change it to `Kubemacpool Opt-modes`?